transit_odp/timetables/utils.py

```diff
@@ -234,3 +235,253 @@ def get_line_description_based_on_direction(row: pd.Series) -> str:
         "antiClockwise": row["inbound_description"],
     }
     return direction_mapping.get(row["direction"], "")
+
+
+def get_vehicle_journey_codes_sorted(
+    df_vehicle_journey_operating: pd.DataFrame,
+) -> List:
+
+    """
+    Get the vehicle journey codes sorted based on the departure time
+    """
+    # Get the unique sorted vehicle journey codes based on the departure time
+    df_vehicle_journey_grouped = df_vehicle_journey_operating.groupby(
```

Comment:
Below code could be considered to return the vehicle journey id order sorted by departure_time

`sorted_df = df_vehicle_journey_operating.groupby('vehicle_journey_id').apply(lambda x: sort_values(by='departure_time'))`

`return df_vehicle_journey_operating['vehicle_journey_id'].unique().tolist()`